extensions/notebook/src/book/bookModel.ts

```diff
@@ -220,12 +220,7 @@ export class BookModel implements azdata.nb.NavigationProvider {
 		return Promise.resolve(result);
 	}
 
-	getPlatformSpecificUri(resource: string): vscode.Uri {
-		if (process.platform === 'win32') {
-			return vscode.Uri.parse(`untitled:${resource}`);
-		}
-		else {
-			return vscode.Uri.parse(resource).with({ scheme: 'untitled' });
-		}
+	getUntitledUri(resource: string): vscode.Uri {
+		return vscode.Uri.parse(`untitled:${resource}`);
 	}
```

Comment:
You tested this and it works as expected on non-windows machines?

Reply:
yep, I tested both on windows and mac and it works as expected.